tests/correctness/std-bit-slice.futil

```diff
@@ -0,0 +1,28 @@
+import "primitives/core.futil";
+import "primitives/memories/comb.futil";
+
+component main(@go go: 1) -> (@done done: 1) {
+  cells {
+    @external mem = comb_mem_d1(32, 1, 1);
+    slice = std_bit_slice(32, 10, 14, 4);
+    pad = std_pad(4, 32);
+  }
+
+  wires {
+    group bit_slice {
+      slice.in = 32'd206256;
```

Comment:
Can we write this number in binary so its more obvious what the slicing is doing?

Reply:
The syntax in Calyx is `32'b0101011` (note the `b` instead of `d`)

Reply:
sure